.github/workflows/nrn-modeldb-ci.yaml

```diff
@@ -28,19 +51,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      matrix:
-        python-version: ["3.9"]
     env:
       DISPLAY: ${{ ':0' }}
-      NEURON_V1: ${{ github.event.inputs.neuron_v1 || 'neuron' }}
-      NEURON_V2: ${{ github.event.inputs.neuron_v2 || 'neuron-nightly' }}
-      MODELS_TO_RUN: ${{ github.event.inputs.models_to_run || '' }}
+      NEURON_V1: ${{ github.event.inputs.neuron_v1 || inputs.neuron_v1 || 'neuron' }}
+      NEURON_V2: ${{ github.event.inputs.neuron_v2 || inputs.neuron_v2 || 'neuron-nightly' }}
+      MODELS_TO_RUN: ${{ github.event.inputs.models_to_run || inputs.neuron_v2 || '' }}
 
     steps:
+        
     - uses: actions/checkout@v3
+      with:
+        repository: ${{ inputs.repo || github.repository }}
         
-    - name: check for Azure drop NEURON_V1 -> ${{ github.event.inputs.neuron_v1 }}
-      if: github.event_name == 'workflow_dispatch' && startsWith(github.event.inputs.neuron_v1, 'https://dev.azure.com/neuronsimulator/')
+    - name: check for Azure drop NEURON_V1 -> ${{ env.NEURON_V1 }}
+      if: startsWith(env.NEURON_V1, 'https://dev.azure.com/neuronsimulator/')
       run: |
         # check URI
         curl -sfSI -X GET '${{ github.event.inputs.neuron_v1 }}'
```

Comment:
Should this also be `${{ env.NEURON_V1 }}` ?

Reply:
it would evaluate like this `${{ startsWith(env.NEURON_V1, 'https://dev.azure.com/neuronsimulator/') }}`, `${{ }}` are optional  here